docs/v2.mdx

```diff
@@ -140,3 +160,4 @@ There are a few, and all are kept track of in [this issue]. We intent to fix all
 of them in future releases.
 
 [this issue]: https://github.com/leancodepl/patrol/issues/1341
+[patrol_v2_article]: https://leancode.pl/blog
```

Comment:
TODO: Change this to the actual blogpost once it's public.

cc @jBorkowska @fylyppo - if I forget, you don't!